src/main.js

```diff
@@ -0,0 +1,12 @@
+import Vue from 'vue'
+import App from './App.vue'
+import 'bootstrap/dist/css/bootstrap.css'
+import 'bootstrap-vue/dist/bootstrap-vue.css'
+import { BootstrapVue } from 'bootstrap-vue'
+
+Vue.use(BootstrapVue)
+Vue.config.productionTip = false
+
+new Vue({
+  render: h => h(App),
+}).$mount('#app')
```

Comment:
not familiar with Vue at all, but it looks similar to React?

Reply:
just another js frame work

Reply:
very similar to React, @leah-h!